inst/model.R

```diff
@@ -0,0 +1,10 @@
+#' Model improvement
+
+# Data
+data <- mtcars
+
+# Fit a regression model to the data
+mod <- lm(mpg ~ wt + hp, data = data)
+
+# Print the model
+mod
```

Comment:
Have you thought about plotting the results of the regression, using `plot()`?

Reply:
Thanks for pointing that out. #6.

Reply:
Thanks for pointing that out. I've made a change here [708bb98](https://github.com/jamesmbaazam/codereviewdemo/pull/4/commits/708bb98058d2de1019ccc8a96c7a2267bad9566e).